lms/resources/_js_config.py

```diff
@@ -363,7 +364,33 @@ def _hypothesis_client(self):
                     "authority": self._authority,
                     "enableShareLinks": False,
                     "grantToken": self._grant_token(api_url),
-                    "groups": [self._context.h_group.groupid(self._authority)],
+                    "groups": self._groups(),
                 }
             ]
         }
+
+    def _groups(self):
+        if self._context.should_use_section_groups:
+            return "$rpc:requestGroups"
+        return [self._context.h_group.groupid(self._authority)]
```

Comment:
This goes into **the Hypothesis client's config object**, not the config for the LMS app's own frontend code. We either send the client the list of one course group's `groupid` as before or, if section groups are in use, we send the client the special string `"$rpc:requestGroups"` which tells it to request the groups over postMessage-JSON-RPC.